packages/browser/src/loader.js

```diff
@@ -1,5 +1,5 @@
 // biome-ignore format: Disabled due to trailing comma not working in IE10/11
```

Comment:
We need to exclude this file, as this is specifically copy-pasted from the sentry repo (where the loader actually lives), and should be exactly the same as there!